build_tools/github/Windows

```diff
@@ -0,0 +1,16 @@
+# Get the Python version of the base image from a build argument
+ARG PYTHON_VERSION
+FROM winamd64/python:$PYTHON_VERSION-windowsservercore
+
+ARG CIBW_TEST_REQUIRES
+ARG WHEEL_NAME
+
+# Install the requirements
+RUN pip install --upgrade pip
+RUN pip install $env:CIBW_TEST_REQUIRES.split(" ")
+
+# Copy and install the Windows wheel
+COPY $WHEEL_NAME $WHEEL_NAME
+RUN pip install $env:WHEEL_NAME
+
+CMD [ "powershell", "-NoProfile", "-Command", "pytest --pyargs sklearn ; python -m threadpoolctl -i sklearn" ]
```

Comment:
This `; python -m threadpoolctl -i sklearn` is probably preventing the pytest return code to be propagated in case of failure.

Reply:
Yeah, I think so. I have decided to directly run these commands in two different container instances so that `pytest` could propagate the error code.

Thank you @ogrisel for your suggestion!